### ISSUE TITLE
Clean button's warning should reflect selected overwrite preference

### ISSUE THREAD

**Issue by bendover22** [bleachbit/bleachbit#215]
If Overwrite files to hide contents is not selected in edit Preferences > General tab,  after  a preview and clicking Clean button, the delete confirmation still says, "Are you sure you want to *permanently* delete the files according to the selected operations?"

This is misleading, if users sometimes turn "wiping" on / off,  or they're not aware of the difference in deleting & securely overwriting / wiping.
The statement indicates overwriting / wiping is enabled when it's not.

If anything, if overwriting pref is disabled, the delete confirmation should state the cleaning method will be simple delete, not "secure overwriting" -  & data could possibly be recovered with enough effort.

Only when Prefs > Overwrite files is selected, should the warning say data will be overwritten (**securely** deleted) so it can't be recovered by most methods.

**Comment by unbob** [comment]
I agree with the OPs comments. Somewhat discouraging that there's been no comments in almost 4 years!
I'm also confused by the context menu 'Shred with BleachBit' -  if 'Overwrite files to hide contents' is not selected in edit Preferences > General tab, does that mean that 'Shred with BleachBit' only deletes the file without overwriting to actually "erase" (Shred) the file? Would be nice if someone could clarify this for me (and possibly others).

**Comment by az0** [comment]
> I'm also confused by the context menu 'Shred with BleachBit' - if 'Overwrite files to hide contents' is not selected in edit Preferences > General tab, does that mean that 'Shred with BleachBit' only deletes the file without overwriting to actually "erase" (Shred) the file? Would be nice if someone could clarify this for me (and possibly others).

The context menu always shreds regardless of the preference set in BleachBit.
